src/server/game/Groups/GroupMgr.cpp

```diff
@@ -91,19 +91,19 @@ void GroupMgr::LoadGroups()
         uint32 oldMSTime = getMSTime();
 
         // Delete all groups whose leader does not exist
-        CharacterDatabase.DirectExecute("DELETE FROM groups WHERE leaderGuid NOT IN (SELECT guid FROM characters)");
+        CharacterDatabase.DirectExecute("DELETE FROM `groups` WHERE leaderGuid NOT IN (SELECT guid FROM characters)");
         // Delete all groups with less than 2 members (or less than 1 for lfg groups)
-        CharacterDatabase.DirectExecute("DELETE groups FROM groups LEFT JOIN ((SELECT guid, count(*) as cnt FROM group_member GROUP BY guid) t) ON groups.guid = t.guid WHERE t.guid IS NULL OR (t.cnt<=1 AND groups.groupType <> 12)");
+        CharacterDatabase.DirectExecute("DELETE FROM `groups` WHERE guid NOT IN (SELECT guid FROM group_member GROUP BY guid HAVING COUNT(guid) > 1)");
```

Comment:
@Kitzunu  can you explain why this change?

Reply:
I change because otherwise we got worldserver startup errors.
I have not actually tested that change myself

Reply:
Without this change, we get this error in worldserver which prevents it from starting:
```
2020-04-16 20:49:34 SQL: DELETE groups FROM groups LEFT JOIN ((SELECT guid, count(*) as cnt FROM group_member GROUP BY guid) t) ON groups.guid = t.guid WHERE t.guid IS NULL OR (t.cnt<=1 AND groups.groupType <> 12)
2020-04-16 20:49:34 ERROR: [1064] You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'groups FROM groups LEFT JOIN ((SELECT guid, count(*) as cnt FROM group_member GR' at line 1
```

Reply:
@Kitzunu yes but what about the logic change here? I don't see this anymore for example: `groups.groupType <> 12`